modular_chomp/code/modules/mob/living/simple_mob/subtypes/vore/carp.dm

```diff
@@ -0,0 +1,37 @@
+/mob/living/simple_mob/animal/space/carp/init_vore()
+	..()
+	var/obj/belly/B = vore_selected
+	B.name = "Stomach"
+	B.desc = "The toothy jaws of the space carp gnash down around your body while its throat opens up to suck you inside. The vicious attack is too swift for you to stop it. You're dragged down its short esophagus, then dumped into its muscular digestive system. Your body curls tight in the cramped confines of its slimy stomach; pushing aside a caustic mixture of other unrecognizable detritus. You realize the soupy chime consists of past explorers who went missing long ago. If you can't push your way back up to freedom, your fate will be the same."
+	B.mode_flags = DM_FLAG_THICKBELLY
+	B.digest_brute = 0.5
+	B.digest_burn = 0.5
+	B.escapechance = 25
+	B.belly_fullscreen = "anim_belly"
+	B.belly_fullscreen_color = "#660021"
+	B.belly_fullscreen_color2 = "#660021"
+	B.fancy_vore = 1
+
+	B.emote_lists[DM_DIGEST] = list(
+		"They say a space carp's belly can digest almost anything. You're going to find out first hand if you stay here for much longer.",
+		"The gastric fluids sizzle ferociously as they ooze across your form.",
+		"A possessive squeeze of the wrinkled walls forces you to scrunch against yourself, as if to say you're not even a bother. This powerful digestive system has processed far more hazardous prey than you.",
+		"The rubbery flesh of the carp's stomach lining constantly pulverizes you from all sides.",
+		"Slowly but surely, everything around you is digested. Organic or not, it's all nourishment to the carp.",
+		"The carp's disgusting innards slowly mix you around with acid. As far as it's concerned, you're nothing but fish food now.",
+		"The walls of the fishy predator push against you from all sides, smushing you into the leftovers of the last adventurer to wind up here.",
```

Comment:
Guh, I meant "The STOMACH walls of the fishy predator" not just "The walls", that sounds silly. Brain borked when I wrote it and skipped a thing.